docs/content/preview/faq/comparisons/cassandra.md

```diff
@@ -76,7 +75,7 @@ With YugabyteDB's native JSON support, developers can now benefit from the struc
 
 ## Operational flexibility
 
-At times, you may need to move your database infrastructure to new hardware or you may want to add a sync/async replica in another region or in public cloud. With YugabyteDB, these operations are simple 1-click intent-based operations that are handled seamlessly by the system in a completely online manner. YugabyteDB's core data fabric and its consensus-based replication model enables the "data tier" to be very agile and recomposable, much like containers/VMs have done for the application or stateless tier.
+At times, you may need to move your database infrastructure to new hardware or you may want to add a synchronous or asynchronous replica in another region or in public cloud. With YugabyteDB, these operations are basic 1-click intent-based operations that are handled seamlessly by the system in a completely online manner. YugabyteDB's core data fabric and its consensus-based replication model enables the "data tier" to be very agile and recomposable, much like containers/VMs have done for the application or stateless tier.
```

Comment:
You may need to move your database infrastructure to new hardware or you may want to add a synchronous or asynchronous replica in another region or in a public cloud. With YugabyteDB, these operations are basic one-click intent-based operations that are handled seamlessly by the system completely online. YugabyteDB's core data fabric and its consensus-based replication model enables the data tier to be very agile and recomposable, similarly to containers' and VMs' handing of the application or stateless tier.